todo_app/data/todo_items.py

```diff
@@ -0,0 +1,31 @@
+import collections
+from turtle import title
+from typing import Collection
+from unittest import result
+import requests
+import json
+import os
+import pymongo
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
Similarly, we shouldn't need this line as Flask automatically loads our environment as long as we've run `poetry run flask run` (and we load it intentionally/separately in production)

Reply:
Removed